website/source/docs/secrets/azure/index.html.md

```diff
@@ -50,10 +54,16 @@ management tool.
     If you are running Vault inside an Azure VM with MSI enabled, `client_id` and
     `client_secret` may be omitted. For more information on authentication, see the [authentication](#authentication) section below.
 
-1. Configure a role. Roles determine the permissions that the service principal
-generated by Vault will have to Azure resources.
+1. Configure a role. A role may be set up with either an existing service principal, or
+a set of Azure roles that will be assigned to a dynamically created service principal.
+
+    To configure a role called "my-role" with an existing service principal:
```

Comment:
Is it intended for this line to be indented? I also notice a bunch of other non-code lines are indented from here thru line 80. If it doesn't make any difference in how the text is displayed, then it's nothing to worry about, but just thought I'd ask.

Reply:
Updated.